paddle/fluid/operators/conditional_block_op.cc

```diff
@@ -31,7 +31,7 @@ class ConditionalOp : public framework::OperatorBase {
   std::vector<const framework::LoDTensor *> InputTensors(
       const framework::Scope &scope) const {
     std::vector<const framework::LoDTensor *> retv;
-    auto xs = Inputs("X");
+    auto xs = Inputs("Params");
```

Comment:
Whether need to run sub block based on the size of `Input(X)` in the raw implementation. But `Input(X)` is the conditional variable.

```c++
    AddInput("X",
             "The conditional variable of this operator. If X is empty, the "
             "whole sub-block will not be executed.")
        .AsDuplicable();
    AddInput("Params", "The input variables of the sub-block.").AsDuplicable();
```

I think this is wrong. It should based on the `Input(Params)`.